boa/src/syntax/parser/statement/declaration/mod.rs

```diff
@@ -0,0 +1,55 @@
+//! <https://tc39.es/ecma262/#sec-declarations-and-the-variable-statement>
```

Comment:
Missing Documentation.
```suggestion
//! Declaration parsing.
//!
//! More information:
//!  - [MDN documentation][mdn]
//!  - [ECMAScript specification][spec]
//!
//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements#Declarations
//! [spec]:https://tc39.es/ecma262/#sec-declarations-and-the-variable-statement
```